src/main/java/net/dv8tion/jda/internal/requests/restaction/pagination/PaginationActionImpl.java

```diff
@@ -358,9 +379,9 @@ protected Route.CompiledRoute finalizeRoute()
         route = route.withQueryParams("limit", limit);
 
         if (last != 0)
-            route = route.withQueryParams(order.getKey(), Long.toUnsignedString(last));
+            route = route.withQueryParams(order.getKey(), getPaginationLastEvaluatedKey(last, this.getLast()));
         else if (order == PaginationOrder.FORWARD)
-            route = route.withQueryParams("after", "0");
+            route = route.withQueryParams("after", getPaginationLastEvaluatedKey(0, this.last));
```

Comment:
Why is one using `getLast` and the other is using `this.last`?

Also, this code is kinad confusing given that there is a localVariable called `last` which is a `long` and a class variable of `last` that is of type `T`.

If the point of making the local variable is for it to be `final` (?) then lets at least call the local variable `lastKey` or `localLastKey` to help break the confusion.